File: app/src/main/java/com/aavens/mindloft/db/RoomContract.java
package com.aavens.mindloft.db;

import android.provider.BaseColumns;

public final class RoomContract {
    public RoomContract() {
    }

    public static abstract class RoomEntry implements BaseColumns {
        public static final String TABLE_NAME = "rooms";
        public static final String COLUMN_NAME_DATE = "date";
        public static final String COLUMN_NAME_TITLE = "title";
    }
}

File: app/src/main/java/com/aavens/mindloft/db/RoomContract.kt
package com.aavens.mindloft.db

import android.provider.BaseColumns

class RoomContract {

    abstract class RoomEntry: BaseColumns {
        companion object {
            val _ID = "_id"
            val TABLE_NAME = "rooms"
            val COLUMN_NAME_DATE = "date"
            val COLUMN_NAME_TITLE = "title"
        }
    }
}
